src/routers/apiRouter.js

```diff
@@ -2,20 +2,36 @@ const userController = require('../controllers/user')
 const placeController = require('../controllers/place')
 const eventController = require('../controllers/event')
 const productController = require('../controllers/product')
-const validateJWT = require('../middleware/validateJWT')
-const validateHeaderToken = require('../middleware/validateHeaderToken')
+
+// const validateJWT = require('../middleware/validateJWT')
```

Comment:
do we want to keep this? If so you should leave a comment explaining

Reply:
I'll just delete it.